docs/strategies/timeout.md

```diff
@@ -16,39 +16,35 @@
 ```cs
 // Add timeout using the default options.
 // See https://www.pollydocs.org/strategies/timeout#defaults for defaults.
-new ResiliencePipelineBuilder()
-    .AddTimeout(new TimeoutStrategyOptions());
+new ResiliencePipelineBuilder().AddTimeout(new TimeoutStrategyOptions());
```

Comment:
Not a hill I'll die on, but I prefer the multi-line formatting as it makes the individual lines less long and breaks things up a bit.

If we're inconsistent I think it would be better to wrap the others, rather than unwrap these.